README.md

```diff
@@ -36,6 +36,70 @@ over 100,000 metric values per second with less than 0.2% overhead.
 
 # Building the OVIS / LDMS source code
 
+## Pre-built containers
+
+You may skip the building process and use the pre-built containers from the
```

Comment:
Are we ready to say here what LDMS version the containers will run on? For example, the latest release. Is there a way for users to query the LDMSD version?